lib/slack/web/api/mixins/conversations.id.rb

```diff
@@ -12,11 +12,23 @@ module Conversations
           #
           # @option options [channel] :channel
           #   Channel to get ID for, prefixed with #.
+          # @option options [integer] :limit
+          #   The page size used for conversations_list calls required to find the channel's ID
           def conversations_id(options = {})
             name = options[:channel]
+            limit = options.fetch(:limit, conversations_id_page_size)
+
             raise ArgumentError, 'Required arguments :channel missing' if name.nil?
 
-            id_for :channel, name, '#', :conversations_list, :channels, 'channel_not_found'
+            id_for(
+              :channel,
+              name,
+              '#',
+              :conversations_list,
+              :channels,
+              'channel_not_found',
+              enum_method_options: { limit: limit }
```

Comment:
I think either we just add one more `limit` parameter to the already long list of arguments or change the signature to just `id_for(options)`? WDYT?

If you think both are a bad idea, at a minimum let's rename `enum_method_options` to just `options` to be future-proof?